scss/_patterns_side-navigation.scss

```
@@ -262,6 +262,7 @@
 
   // Styles for markup in raw HTML docs variant
   .p-side-navigation--raw-html {
+    // stylelint-disable selector-max-type
```

We could add a comment explaining why it is fine here to disable the rule

```suggestion
    // stylelint-disable selector-max-type -- we support raw HTML markup for discourse-generated side nav
```